cordova-lib/src/cordova/plugin.js

```diff
@@ -157,6 +157,7 @@ module.exports = function plugin(command, targets, opts) {
 
                             if (missingVariables.length) {
                                 shell.rm('-rf', pluginInfo.dir);
+                                events.emit('verbose', 'Removing ' + pluginInfo.dir + ' due to installation failure');
                                 var msg = 'Variable(s) missing (use: --variable ' + missingVariables.join('=value --variable ') + '=value).';
```

Comment:
> plugin isn't getting removed since it hasn't been installed

Right, but the previous message in console states something like 'Copying plugin ../source/path => plugin/destination path' (see [fetch.js#L352](https://github.com/apache/cordova-lib/blob/master/cordova-lib/src/plugman/fetch.js#L352)) so we just crarifying that we've copied the folder but now removing it due to restore failure

> replacing it by events.emit(msg), with msg defined at L161

This is already emitted at [restore-util#R127](https://github.com/apache/cordova-lib/pull/420/files#diff-e9221ac2a69074d78c7fb9a8e6b9d4d6R127), why we need to duplicate that?

![image](https://cloud.githubusercontent.com/assets/3857604/14387763/8923e75a-fdb3-11e5-80c0-3f3d7b133686.png)